Markdown/cmm-2.0.14/system-operators/2-installation.md

```diff
@@ -1069,12 +1069,24 @@ $ python3.6 -m pip install --no-index --find-links="/opt/monasca-ui" monasca-ui=
  
 Pls. replace `x` in monasca-ui version the following way:  
```

Comment:
I would write   
"determine 'x' to be used in monasca-ui version..."  
" instead of  
"replace 'x' in monasca-ui version"